### PR TITLE
Payment chooser page

### PR DESCRIPTION
**Pull request by nuxzero**
### Objective

The create the payment chooser page to able users choose a payment method that wants to make a payment. 

### Description of Changes

In this PR created the `PaymentCreatorActivity`  to control the flow of creating a payment from the preferred payment method. And created `PaymentChooserFragment` as a starting page for choose different payment methods. 

Also, provided the foundation of building pages in the payment creator flow.

### QA

All instrumented tests that involve to this PR shall be passed. However, some of the unit test there is a failed test still investigating.

Screenshot_1568010343

Screenshot_1568010432

### CODE REVIEW

**Comment by pitiphong-p** [comment]
Can I have a look at all of the screens?

**Comment by nuxzero** [comment]
Sorry, there is only payment chooser page for this PR. From the demo it on other branch.

**Comment by nuxzero** [comment]
I breaked it down to small PR. So, other pages will come in future PR. 

Sorry for inconvenience.

**Comment by pitiphong-p** [comment]
Oh I got it. OK and sorry for asking that

File: sdk/src/sharedTest/java/co/omise/android/ui/PaymentChooserFragmentTest.kt [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        )
+        val capability = Capability(
+                paymentMethods = paymentMethods
+        )
+
+        fragment = PaymentChooserFragment.newInstance(capability).apply {
+            navigation = mockNavigation
+        }
+
+        intending(hasComponent(hasClassName(TestFragmentActivity::class.java.name)))
```
Comment (Namiii) [inline sdk/src/sharedTest/java/co/omise/android/ui/PaymentChooserFragmentTest.kt]: Can you tell me what this line does?

File: sdk/src/main/java/co/omise/android/ui/PaymentChooserFragment.kt [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuInflater
+import android.view.MenuItem
+import co.omise.android.R
+import co.omise.android.models.Capability
+import co.omise.android.models.PaymentMethodType
+import co.omise.android.models.method
+
+class PaymentChooserFragment : OmiseListFragment<PaymentChooserItem>() {
```
Comment (Namiii) [inline sdk/src/main/java/co/omise/android/ui/PaymentChooserFragment.kt]: Great setup here 👌

File: sdk/src/sharedTest/java/co/omise/android/ui/PaymentChooserFragmentTest.kt (hunk 1 above)
Comment (nuxzero) [reply sdk/src/sharedTest/java/co/omise/android/ui/PaymentChooserFragmentTest.kt]: This is a stubbing intent responses.

**Pull request merged**